ts/components/InstabugButtons.tsx

```diff
@@ -19,25 +19,49 @@ type Props = ReturnType<typeof mapStateToProps> &
   OwnProps &
   ReturnType<typeof mapDispatchToProps>;
 
-class InstabugButtonsComponent extends React.PureComponent<Props, {}> {
+type State = {
+  instabugReportType: "bug" | "chat" | "undefined";
+};
+
+class InstabugButtonsComponent extends React.PureComponent<Props, State> {
   private handleIBChatPress = () => {
-    this.props.dispatchIBReportOpen("chat");
-    Chats.show();
+    this.setState({ instabugReportType: "chat" });
+    this.props.dispatchIBReportOpen(this.state.instabugReportType);
```

Comment:
from documentation
> setState() does not immediately mutate this.state but creates a pending state transition. Accessing this.state after calling this method can potentially return the existing value. There is no guarantee of synchronous operation of calls to setState and calls may be batched for performance gains.

So you are no guarantee **this.state.instabugReportType** has the value set in the previous line.
Use a const instead

Reply:
Done